src/map/battle.cpp

```diff
@@ -805,6 +808,11 @@ int battle_calc_cardfix(int attack_type, struct block_list *src, struct block_li
 					}
 					if (s_defele != ELE_NONE)
 						ele_fix += tsd->indexed_bonus.magic_subdefele[s_defele] + tsd->indexed_bonus.magic_subdefele[ELE_ALL];
+#ifndef RENEWAL
```

Comment:
Do we care about adding support to pre-renewal when the end goal is to remove renewal skills from the pre-re databases?

Reply:
That's from the original legacy. Skills and SCs are common by default in src. Even if there is some weird behavior like in this PR, separating pre-re and re in src will make things harder for people who mix the modes. Should we change our way and separate them like the DB?